src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.UnknownUnix.cs

```diff
@@ -9,7 +9,7 @@ public partial class Process : IDisposable
         /// Creates an array of <see cref="Process"/> components that are associated with process resources on a
         /// remote computer. These process resources share the specified process name.
         /// </summary>
-        public static Process[] GetProcessesByName(string processName, string machineName)
+        public static Process[] GetProcessesByName(string? processName, string machineName)
```

Comment:
The warning should be suppressed for iOS tvOS too: https://github.com/dotnet/runtime/blob/5fda426afd6cb2ca3c226d4f471bb8bed37cbfbc/src/libraries/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj#L8